### PR TITLE
Added configuration parameters for mode, user and group of LDAPI socket

### LINKED ISSUE #83: Possibility to change rights and ownership of LDAPI socket
By default the socket is created with 777 and root:root:
```
# ll /var/run/slapd/ldapi 
srwxrwxrwx 1 root root 0  4 sept. 10:06 /var/run/slapd/ldapi
```

It could be interesting to be able to configure it.

OpenLDAP propose a `x-mod` option that can be found in man page:
```

              The listener permissions are indicated by "x-mod=-rwxrwxrwx", "x-mod=0777" or "x-mod=777", where any of the "rwx" can be "-" to suppress the  related  permission,  while
              any  of the "7" can be any legal octal digit, according to chmod(1).  The listeners can take advantage of the "x-mod" extension to apply rough limitations to operations,
              e.g. allow read operations ("r", which applies to search and compare), write operations ("w", which applies to add, delete, modify and modrdn),  and  execute  operations
              ("x",  which means bind is required).  "User" permissions apply to authenticated users, while "other" apply to anonymous users; "group" permissions are ignored.  For ex‐
              ample, "ldap:///????x-mod=-rw-------" means that read and write is only allowed for authenticated connections, and bind is required for all operations.  This feature  is
              experimental, and requires to be manually enabled at configure time.
```

But this is experimental.

I think it could be easier to add parameters in slapd-cli to do a chown and chmod if they are set.

### PR DESCRIPTION
**Pull request by armfem**
Closes #83

### CODE REVIEW

File: slapd-cli [hunk 1]
```diff
@@ -435,6 +438,22 @@ start_slapd() {
 			$SLAPD_BIN -h "$SLAPD_SERVICES" $SLAPD_PARAMS
 		fi
 		sleep 1
+
```
Comment (davidcoutadeur) [inline slapd-cli:441]: Could your remove the blanks at ends of lines 441 and 442?

**Review by davidcoutadeur — CHANGES_REQUESTED** [review]
This PR is quite good! Thanks for your contribution @armfem 

2 small remarks:
- could you remove the extra blanks at the end of lines?
- could you add the number of the issue in the commit message? (#83)

**Comment by coudot** [comment]
> could you add the number of the issue in the commit message? (https://github.com/ltb-project/slapd-cli/issues/83)

Not required from my point of view, as the commit are linked to a PR which will be associated to the issue

**Comment by davidcoutadeur** [comment]
> > could you add the number of the issue in the commit message? (#83)
> 
> Not required from my point of view, as the commit are linked to a PR which will be associated to the issue

I agree, it is not mandatory, but I find it more convenient.

**Pull request merged**